src/assets/icons/OutlineExclamationCircle.tsx

```diff
@@ -0,0 +1,17 @@
+import { Icon, IconProps } from "@chakra-ui/react";
+
+import colors from "../../style/colors";
+
+export const OutlineExclamationCircleIcon: React.FC<IconProps> = props => (
+  <Icon
+    width="16px"
+    height="16px"
+    fill={colors.white}
+    viewBox="0 0 1024 1024"
```

Comment:
these numbers look quite odd. where'd get them from?

Reply:
Original react icons have this numbers: https://react-icons.github.io/react-icons/search/#q=AiOutlineExclamationCircle

<img width="1227" alt="Screenshot 2024-03-13 at 10 49 07" src="https://github.com/trilitech/umami-v2/assets/150050388/0a9ec107-e09c-49e2-95e8-19d52daab7a6">

<img width="650" alt="Screenshot 2024-03-13 at 10 50 13" src="https://github.com/trilitech/umami-v2/assets/150050388/d63a6168-5677-4008-bfe1-f0180fad467c">